html/index.html

```diff
@@ -14,6 +14,15 @@
   <script src="app.js"></script>
 </head>
 <body>
-<div class="loader">Lade<br /><span class="spinner"></span><br />Karten &amp; Knoten...</div>
+<div class="loader">
+  <p>
+    Lade<br />
+    <span class="spinner"></span><br />
```

Comment:
Magst nach den `<br />` in einer neuen zeile holen?
dahinten übersieht man in zu leicht, entweder alle untertags von `<p>` in einer Zeile oder nicht ...